CommandLine/emake/Main.cpp

```diff
@@ -34,6 +36,17 @@ static std::string tolower(const std::string &str) {
 
 int main(int argc, char* argv[])
 {
+  std::string ENIGMA_DEBUG = (std::getenv("ENIGMA_DEBUG") ? std::getenv("ENIGMA_DEBUG") : "");
+  if (ENIGMA_DEBUG == "TRUE") {
```

Comment:
Eugh. This makes it totally impossible to print anything in this process. We should revisit this when emake is used for more than just the CI.

Reply:
it does not. you can still print via cout here

Reply:
Read this backward. Ignore me.